go.sum

```diff
@@ -106,6 +106,8 @@ github.com/cilium/ebpf v0.0.0-20191113100448-d9fb101ca1fb/go.mod h1:MA5e5Lr8slmE
 github.com/cilium/hubble v0.0.0-20200304205543-565cbe60f7fb/go.mod h1:GYdTLWzzpzgp2ptyEp0LccPotGo4BxPX9SJRGL4pI5c=
 github.com/cilium/hubble v0.0.0-20200320161706-5faac2a81350 h1:Emt6IBafrgi95wUCKJdspH6vuTx9mmpwOEEwo61j5U4=
```

Comment:
I'd suggest to run `go mod tidy` to clean stray entries from `go.sum`.

Reply:
Good catch! Done.